dev/com.ibm.ws.concurrent.persistent_fat_compat/publish/servers/com.ibm.ws.concurrent.persistent.fat.compat/.gitignore

```diff
@@ -0,0 +1,2 @@
+/derby
```

Comment:
this file can be ignored, and in general all .gitignore files can be removed from FATs as we bring them over